src/main/kotlin/dev/arbjerg/ukulele/command/SkipCommand.kt

```diff
@@ -18,7 +18,11 @@ class SkipCommand : Command("skip", "s") {
     }
 
     private fun CommandContext.skipNext() {
+        val repeatTrackPreSkip = player.repeatTrack
         printSkipped(player.skip(0..0))
+        if (repeatTrackPreSkip && !player.repeatTrack) {
+            reply("Repeat track was turned off on skip.")
+        }
```

Comment:
Displays a message that repeat track is disabled if it was enabled prior to skipping and is disabled after skip.